src/components/MemberProgress/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { convertDate } from '../utilis';
+const MemberProgress = ({ currentMember: member }) => {
+  let id = 1;
```

Comment:
it's redundant

Reply:
deleted variable id:
```suggestion
  {member.progress.map((item, i) => {
            return (
              <tr key={item.id}>
                <td>{++i}</td>
```